src/Bitcoin/Transaction.cpp

```diff
@@ -8,9 +8,11 @@
 #include "SegwitAddress.h"
 #include "SignatureVersion.h"
 #include "SigHashType.h"
+#include "HexCoding.h"
 
 #include "../BinaryCoding.h"
 
+#include <boost/operators.hpp>
```

Comment:
why?

Reply:
No reason, this was imported by the IDE :) removed